client/platform/web-girder/views/AnnotationTags.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import {
+  defineComponent, ref,
+} from '@vue/composition-api';
+import {
+  useAnnotationTag,
+  useAnnotationTags,
+  useHandler,
+} from 'vue-media-annotator/provides';
+
+export default defineComponent({
+  name: 'AnnotationTags',
+  description: 'Annotation Tags',
+
+  setup() {
+    const currentTag = useAnnotationTag();
+    const tags = useAnnotationTags();
+    const newTag = ref('');
+    const validForm = ref(false);
+    const { save, tagChange } = useHandler();
+    const selectedTag = (tag: string) => {
+      tagChange(tag);
+    };
+
+    const addTag = async () => {
+      await save(newTag.value);
+      tagChange(newTag.value);
+    };
+
+    return {
+      currentTag,
+      selectedTag,
+      tags,
+      newTag,
+      addTag,
+      validForm,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <p> Available Tags</p>
```

Comment:
```suggestion
        <div class="px-4 pt-2 text-body-1"> Available Tags</div>
```

This would align the text and improve the spacing.